controllers/UserController.js

```diff
@@ -0,0 +1,203 @@
+const express = require("express"),
+    router = express.Router(),
+    db = require('../models'),
+    isAuthenticated = require("../config/middleware/isAuthenticated"),
+    passport = require("../config/passport");
+
+//const { WebhookClient } = require('dialogflow-fulfillment');
+
+class UserRegistrationController {
+  
+  newUserIntent(agent, body) {
+    console.log("Storing new User in DB");
+    let fName = agent.parameters["user-given-name"];
+    let cNum = body.From.toSring();
+
+    if(db.userWA.findAll({where: {phone_number: cNum}}).length > 0) return "Phone Number already Exists";
```

Comment:
Also, .length>0 doesn't work. Please test your code. 
Also, for such a function, refer to the current master branch's way of checking user.